gallery/src/styles/Nivo/NivoBarExample.tsx

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { ResponsiveBar, BarLegendProps } from '@nivo/bar';
+import { NIVO_COLORS } from '@sonatype/react-shared-components';
+import { rawBarData } from './data';
+
+const LEGENDS: BarLegendProps[] = [
+  {
+    dataFrom: 'indexes',
+    anchor: 'bottom-right',
+    direction: 'column',
+    itemWidth: 100,
+    itemHeight: 20,
+    translateX: 140
+  }
+];
+
+const barData = rawBarData.map(([id, value]) => ({ id, value }));
+
+const MyResponsiveBar = () => (
```

Comment:
I'm not seeing a reason for this to be a separate component rather than just being part of `NivoBarExample`.

Reply:
> I'm not seeing a reason for this to be a separate component rather than just being part of `NivoBarExample`.

Updated here: 716b4075f68e484d0aa313a771cdb718a2a37bf5